### PR TITLE
2022-08-25 Console release

### PR DESCRIPTION
**Pull request by antonyc**

### CODE REVIEW

**Review by ololobus — DISMISSED** [review]
Let's be consistent with:
- lowercase letter after `:`
- dot at the end of text

File: content/docs/release-notes.md [hunk 1]
```diff
@@ -2,6 +2,26 @@
 title: Release Notes
 ---
 
+## 2022-08-25 Console release
+
+### Bug fixes
+
+- UI: Correct .pgpass instructions [#1825](https://github.com/neondatabase/neon/issues/1825)
```
Comment (stepashka) [inline content/docs/release-notes.md:9]:
```suggestion
- UI: fix .pgpass instructions [#1825](https://github.com/neondatabase/neon/issues/1825).
```

File: content/docs/release-notes.md [hunk 2]
```diff
@@ -2,6 +2,26 @@
...
+
+- UI: add validation messages for creating a project form.
+
+- UI: fix the dead links in the onboarding
+
+- UI: limited preview and branching email updates
```
Comment (stepashka) [inline content/docs/release-notes.md:15]:
```suggestion
- UI: update the emails for requesting branching and support.
```

File: content/docs/release-notes.md [hunk 3]
```diff
@@ -2,6 +2,26 @@
...
+
+- UI: add validation messages for creating a project form.
+
+- UI: fix the dead links in the onboarding
+
+- UI: limited preview and branching email updates 
+
+### What's new
+
+- UI: Table viewer
```
Comment (stepashka) [inline content/docs/release-notes.md:19]:
```suggestion
- UI: added Tables page for exporing the Project schema and data.
```

File: content/docs/release-notes.md [hunk 4]
```diff
@@ -2,6 +2,26 @@
...
+
+- UI: add help center menu 
+
+- Control Plane: Select safekeepers from different Availability Zones for a new compute project.
```
Comment (stepashka) [inline content/docs/release-notes.md:23]:
```suggestion
- Control Plane: pick the safekeepers from different Availability Zones for the new Projects.
```

File: content/docs/release-notes.md [hunk 5]
```diff
@@ -2,6 +2,26 @@
...
 
+## 2022-08-25 Console release
+
+### Bug fixes
+
+- UI: Correct .pgpass instructions [#1825](https://github.com/neondatabase/neon/issues/1825)
+
+- UI: add validation messages for creating a project form.
+
+- UI: fix the dead links in the onboarding
```
Comment (stepashka) [inline content/docs/release-notes.md:13]:
```suggestion
- UI: fix the dead links in the onboarding.
```

**Review by stepashka — COMMENTED** [review]
Suggested letter case, punctuation and wording changes to ensure style consistency in the release notes.

File: content/docs/release-notes.md [hunk 6]
```diff
@@ -2,6 +2,26 @@
...
+
+- UI: fix the dead links in the onboarding.
+
+- UI: update the emails for requesting branching and support.
+
+### What's new
+
+- UI: added Tables page for exporing the Project schema and data.
+
+- UI: add help center menu
```
Comment (ololobus) [inline content/docs/release-notes.md:21]:
```suggestion
- UI: add help center menu.
```

**Review by ololobus — APPROVED** [review]

**Pull request merged**